app/views/posts/index.html.erb

```diff
@@ -1,6 +1,37 @@
-<h1>Welcome to post index page</h1>
-<ul>
-  <% @posts.each do |post| %>
-    <li><%= post.title %></li>
-  <% end %>
-</ul>
+<div class='user_show_container'> 
+  <div class='headline_container'>
+    <div class="image_container" style="width: 15rem; height: 15rem;">
+      <% if @user.photo %>
+        <img src="<%= @user.photo %>" class="" >
+      <% else %>
+        <p> No image found </p>
+      <% end %>
+    </div>
+    <div class="content_show_container">
+      <h1 class="">User Name: <%= @user.name %></h1>  
+      <h6 class="">Number of posts: <span><%= @user.posts_counter %></span></h6>
+    </div>
+  </div>
+
+  <div class="user_posts">
+    <h2>Posts</h2>
+    <% @user.recent_posts.each do |post| %>
+      <div class='posts_list'>
+        <h3>Title: <%= post.title %></h3>
```

Comment:
- [ ] As suggested by the previous reviewer and also as per design requirements. every post title on the Post index page should be a link that takes the user to the Post show page of that respected post page. 
right now user cannot click on the individual post title so consider fixing this bug by converting post titles to link so that it takes user to Post show page of that post.